src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -960,7 +980,7 @@ static void applyConcurrentValidations(Config config)
         else if (config.concurrent_validations > config.concurrent_compactors && !allowUnlimitedConcurrentValidations)
         {
             throw new ConfigurationException("To set concurrent_validations > concurrent_compactors, " +
-                                             "set the system property cassandra.allow_unlimited_concurrent_validations=true");
+                                             "set the system property " + ALLOW_UNLIMITED_CONCURRENT_VALIDATIONS.getKey() + "=true");
```

Comment:
maybe including "-D" would be OK? 

`set the system property -D" + ALLOW_UNLIMITED_CONCURRENT_VALIDATIONS.getKey() + "=true"`

Reply:
Fixed.